src/utils/config.py

```diff
@@ -25,6 +27,7 @@ def get_config() -> Config:
     """
 
     config_file = os.environ.get("MODELS_CONFIG_FILE_PATH", "config/config.yml")
+    get_logger("config").info(f"Loading models config from: {config_file}")
```

Comment:
nitpick: maybe init the logger once outside the class after the imports. We may use it for other functions.

Reply:
updated